corehq/apps/reports/static/reports/javascripts/config.dataTables.bootstrap.js

```diff
@@ -1,6 +1,14 @@
-// datatable configuration.
+(function (factory) {
```

Comment:
Boilerplate to handle both "old" and "new" worlds: if `define` exists, we can use RequireJS, otherwise fall back to `hqDefine`.